### PR TITLE
Add jsonschema Python implementation

### PR DESCRIPTION
**Pull request by michaelmior**

### CODE REVIEW

File: implementations/jsonschema/version.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+
+# Extract the version of the package (boon) from Cargo.toml
```
Comment (tony-go) [inline implementations/jsonschema/version.sh:6]:
```suggestion
# Extract the version of the package (jsonschema) from uv.lock
```

Comment (tony-go) [reply implementations/jsonschema/version.sh:6]: I wonder if `pyproject.toml` would not work better? lock files are generally speaking a mess! haha

File: pyproject.toml [hunk 2]
```diff
@@ -5,4 +5,5 @@ description = "Add your description here"
 requires-python = ">=3.12"
 dependencies = [
     "seaborn>=0.13.2",
+    "jsonschema>=4.23.0",
```
Comment (tony-go) [inline pyproject.toml:8]: Also, I wonder if we should not split the python project for the implementation from the one we use for plots.

Then we could encapsulte lockfiles and configuration files in each implementations folders.

**Review by jviotti — APPROVED** [review]

File: implementations/jsonschema/version.sh (hunk 1 above)
Comment (michaelmior) [reply implementations/jsonschema/version.sh:6]: @tony-go Sure, that's fine if we want to specify the version in the toml file. Right now I haven't. Either way works for me.

File: pyproject.toml (hunk 2 above)
Comment (michaelmior) [reply pyproject.toml:8]: I thought about that, but I figured I'd delay that until if and when we run into conflicts. But I'm fine making a separate environment if that feels cleaner.

Comment (tony-go) [reply pyproject.toml:8]: I did on a seperate PR for node: https://github.com/sourcemeta-research/jsonschema-benchmark/pull/20

So nw :)

Comment (michaelmior) [reply pyproject.toml]: Made those changes here too.

**Pull request merged**